include/bsp/iotc_bsp_tls.h

```diff
@@ -19,25 +19,24 @@
 
 /**
  * @file iotc_bsp_tls.h
- * @brief Implement Transport Layer Security (TLS).
+ * @brief Implements Transport Layer Security (TLS).
  *
- * Reference implementations are in the <a href="~src/bsp/tls"><code></code></a>
- * directory. All TLS functions are non-blocking operations.
+ * @details Reference implementations are in the
+ * <a href="~/src/bsp/tls"><code></code></a> directory. All TLS functions are
```

Comment:
this link isn't visible, pls check

Reply:
Done. Also fixed other relative links throughout the BSP reference

Reply:
ping